platform_versioned_docs/version-24.1.0/enterprise/configuration/configtables/generic_config_aws.yml

```diff
@@ -30,6 +30,11 @@
     Your Seqera system administrator contact email.
   Value:                '`tower@your-company.com`'
 - 
+  AWS Parameter Store:    '`{prefix}/tower/disable-email`'
```

Comment:
You sure about this? My (functional) tower.yml is `tower.auth.disable-email`. You are missing the `auth` part in your prefix.

Generally the PS prefix path follows the structure of the tower.yml.

Reply:
Fixed.